OpacPica-Plugin/pom.xml

```diff
@@ -32,6 +32,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <parent.basedir>${basedir}/../</parent.basedir>
         <checkstyle.config.location>${parent.basedir}/config/checkstyle.xml</checkstyle.config.location>
+        <phase.prop>none</phase.prop>
```

Comment:
you don't need this property in the child poms, because you already defined it in the parent pom. Only overwrite it, where it should be executed.